scripts/prepare-core.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+
+var dir = path.resolve(process.cwd(), 'core');
+
+if (!fs.existsSync(dir)) {
+  fs.mkdirSync(dir);
+}
+
+const pathToPkgJson = path.resolve(dir, 'package.json');
+if (fs.existsSync(pathToPkgJson)) {
+  fs.unlinkSync(pathToPkgJson);
+}
+
+const contents = JSON.stringify({
+  name: 'urql-core',
+  version: '1.5.0',
+  description:
```

Comment:
May make sense to set the private field to true here so we don't actually publish this accidentally. Does it make sense to use the name "@urql/core" anyway and publish a stub package to block the name?

Reply:
Lets maybe also copy some of the fields from the main package json?